src/commonTest/kotlin/fr/acinq/lightning/channel/states/SpliceTestsCommon.kt

```diff
@@ -1187,9 +1467,10 @@ class SpliceTestsCommon : LightningTestSuite() {
             val (alice1, actionsAlice1) = alice.process(ChannelCommand.MessageReceived(commitSigBob))
             assertTrue(actionsAlice1.isEmpty())
             val (bob1, actionsBob1) = bob.process(ChannelCommand.MessageReceived(commitSigAlice))
+            val incomingHtlcs = bob1.commitments.latest.localCommit.spec.htlcs.incomings()
             when {
-                bob1.staticParams.useZeroConf -> assertEquals(actionsBob1.size, 4)
-                else -> assertEquals(actionsBob1.size, 3)
+                bob1.staticParams.useZeroConf -> assertEquals(actionsBob1.size, 4 + incomingHtlcs.size)
```

Comment:
Instead of checking the number of actions, we should rather check that we have the right actions for pending HTLCs.